react-backend/routes/cbauthentication/cb_password_reset.js

```diff
@@ -24,16 +24,16 @@ router.post('/', (req, res, next) => {
           .then(token =>
             Promise.all([
               pwdTokenAdd(token, tokenExpire, formEmail),
-              sendResetEmail(formEmail, token),
+              sendResetEmail(formEmail, token)
             ])
           )
           .then(() => res.send(exists))
           .catch(err => {
             console.log('Error sending email:', err);
-            res.send('failed');
+            res.status(501).send('failed');
```

Comment:
`501` is "not implemented". If you wanted a generic server error, that would be `500`. Although in this case, a `502` (bad gateway) is probably more appropriate.